installers/dhall/linux.dhall

```diff
@@ -1,25 +1,34 @@
--- XXX: deal with XDG_DATA_HOME generalisation -- pass dataDir as an argument?
+\(cluster : ./cluster.type)      ->
 let dataDir = "$HOME/.local/share/Daedalus/"
+    -- XXX: deal with XDG_DATA_HOME being an actual variable -- pass dataDir as an argument?
+    --    ..or, maybe installation data is a better solution?
 in
-{ name      = "linux"
-, nodeArgs  =
-  { keyfile          = dataDir ++ "Secrets-1.0/secret.key"
-  , logsPrefix       = dataDir ++ "Logs"
-  , updateLatestPath = dataDir ++ "FIXME"
-  , walletDBPath     = dataDir ++ "Wallet-1.0"
+{ name      = "macos64"
+    -- XXX: this is an ugly workaround for the lack of a linux configuration key in configuration.yaml
```

Comment:
Do we want a `cardano-sl` PR for that?

We don't have Linux update machinery to inform us what would the naming be -- yet.

Reply:
totally overlooked that, and that will need to be fixed before the update testing can go any further

Reply:
@cleverca22, https://github.com/input-output-hk/cardano-sl/pull/2748